run/cardano2john.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+#
+# Export Cardano's legacy secret.key to John The Ripper format
+#
+# This software is Copyright (c) 2022, Pal Dorogi (ilap) <pal dot dorogi at gmail.com>
+# and it is hereby released to the general public under the following terms:
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted.
+#
+# Requireements: pip3 at OS level
```

Comment:
Typo.

Also, does this not work with pip3 at user level? Why not?

Reply:
Fixed, sorry it would run on user level too.